tsstats/tests/test_ident_map.py

```diff
@@ -24,3 +25,33 @@ def test_ident_map(identmap_clients):
     assert clients['5'] == cl
     assert clients['UID1'] == uidcl
     assert clients['UID5'] == uidcl
+
+
+@pytest.fixture(scope="module")
+def pretty_identmap():
+    return [
+        {'primary_id': '1', 'alternate_ids': ['3', '6']},
+        {'primary_id': '4', 'alternate_ids': ['9', '42']}
+    ]
+
+
+@pytest.mark.parametrize('test_input,expected', [
+    (
+        [
+            {'primary_id': '1', 'alternate_ids': ['3', '6']},
+            {'primary_id': '4', 'alternate_ids': ['9', '42', '23']}
+        ],
+        (('3', '1'), ('6', '1'), ('9', '4'), ('42', '4'), ('23', '4'))
```

Comment:
Yeah, you're absolutely right, there's no need for a mutable type here.
I do prefer a `list` in cases where I iterate over a store with multiple objects, personally, but you can just leave it as a `tuple` here 👍